pkg/http_endpoint/authentication/jwt_authentication.go

```diff
@@ -0,0 +1,106 @@
+package httpAuth
+
+import (
+	"github.com/auth0/go-jwt-middleware"
+	"github.com/dgrijalva/jwt-go"
+	"net/http"
+	"time"
+	"visualization-api/pkg/http_endpoint/common"
+)
+
+const contextJWTProperty = "AuthToken"
+
+// JWTIssuer
+const tokenIssuer = "visualization-api"
+
+// CustomClaims defines what data would be stored in jwt token
+type CustomClaims struct {
+	IsAdmin   bool   `json:"is_admin"`
+	ProjectID string `json:"organizationId"`
+	jwt.StandardClaims
+}
+
+// JWTTokenFromParams creates jwt token given claims data
+func JWTTokenFromParams(secret string, isAdmin bool, projectID string,
+	expiresAt time.Time) (string, error) {
+
+	claims := CustomClaims{
+		isAdmin,
+		projectID,
+		jwt.StandardClaims{
+			ExpiresAt: expiresAt.Unix(),
```

Comment:
This data is encoded inside JWT token and is not visible to use, so I've chosen shortest format possible.
Real http response has format, that corresponds to spec